File: app/src/main/java/ru/nikshlykov/englishwordsapp/ui/statistics/StatisticsViewModel.java
package ru.nikshlykov.englishwordsapp.ui.statistics;

import android.app.Application;

import androidx.annotation.NonNull;
import androidx.lifecycle.AndroidViewModel;
import androidx.lifecycle.MutableLiveData;

import java.util.ArrayList;

import javax.inject.Inject;

import ru.nikshlykov.englishwordsapp.db.WordsRepository;
import ru.nikshlykov.englishwordsapp.db.repeat.Repeat;

public class StatisticsViewModel extends AndroidViewModel {

    private WordsRepository wordsRepository;

    private MutableLiveData<ArrayList<Repeat>> allRepeatsMutableLiveData;

    @Inject
    public StatisticsViewModel(@NonNull Application application, WordsRepository wordsRepository) {
        super(application);
        this.wordsRepository = wordsRepository;

        allRepeatsMutableLiveData = new MutableLiveData<>();
    }

    public void loadStatistics(){

    }

    public MutableLiveData<ArrayList<Repeat>> getAllRepeatsMutableLiveData() {
        return allRepeatsMutableLiveData;
    }
}

File: app/src/main/java/ru/nikshlykov/englishwordsapp/ui/viewmodels/StatisticsViewModel.kt
package ru.nikshlykov.englishwordsapp.ui.viewmodels

import android.app.Application
import androidx.lifecycle.AndroidViewModel
import androidx.lifecycle.MutableLiveData
import ru.nikshlykov.data.database.models.Repeat
import java.util.*

class StatisticsViewModel(application: Application) :
  AndroidViewModel(application) {
  val allRepeatsMutableLiveData: MutableLiveData<ArrayList<Repeat>> = MutableLiveData()

  fun loadStatistics() {}
}